articles/industrial-iot/tutorial-configure-industrial-iot-components.md

```diff
@@ -48,33 +48,49 @@ output of deployment script or reset the password
 * IoT Hub → IoT Edge → \<DEVICE> → Set Modules → OpcPublisher (for standalone OPC Publisher operation only)
 
 
-## Configuration options
+## OPC Publisher 2.8.2 Configuration options for orchestrated mode
 
-|Configuration Option (shorthand/full name)    |    Description   |
-|----------------------------------------------|------------------|
-pf/publishfile |The filename to configure the nodes to publish. If this option is specified, it puts OPC Publisher into standalone mode.
-lf/logfile |The filename of the logfile to use.
-ll/loglevel |The log level to use (allowed: fatal, error, warn, info, debug, verbose).
-me/messageencoding |The messaging encoding for outgoing messages allowed values: Json, Uadp
-mm/messagingmode |The messaging mode for outgoing messages allowed values: PubSub, Samples
-fm/fullfeaturedmessage |The full featured mode for messages (all fields filled in). Default is 'true', for legacy compatibility use 'false'
-aa/autoaccept |The publisher trusted all servers it's a connection to
-bs/batchsize |The number of OPC UA data-change messages to be cached for batching.
-si/iothubsendinterval |The trigger batching interval in seconds.
-ms/iothubmessagesize |The maximum size of the (IoT D2C) message.
-om/maxoutgressmessages |The maximum size of the (IoT D2C) message egress buffer.
-di/diagnosticsinterval |Shows publisher diagnostic info at the specified interval in seconds (need log level info). -1 disables remote diagnostic log and diagnostic output
-lt/logflugtimespan |The timespan in seconds when the logfile should be flushed.
-ih/iothubprotocol |Protocol to use for communication with the hub. Allowed values: AmqpOverTcp, AmqpOverWebsocket, MqttOverTcp, MqttOverWebsocket, Amqp, Mqtt, Tcp, Websocket, Any
-hb/heartbeatinterval |The publisher is using this as default value in seconds for the heartbeat interval setting of nodes without a heartbeat interval setting.
-ot/operationtimeout |The operation timeout of the publisher OPC UA client in ms.
-ol/opcmaxstringlen |The max length of a string opc can transmit/receive.
-oi/opcsamplinginterval |Default value in milliseconds to request the servers to sample values
-op/opcpublishinginterval |Default value in milliseconds for the publishing interval setting of the subscriptions against the OPC UA server.
-ct/createsessiontimeout |The interval the publisher is sending keep alive messages in seconds to the OPC servers on the endpoints it's connected to.
-kt/keepalivethresholt |Specify the number of keep alive packets a server can miss, before the session is disconnected.
-tm/trustmyself |The publisher certificate is put into the trusted store automatically.
-at/appcertstoretype |The own application cert store type (allowed: Directory, X509Store).
+The following OPC Publisher configuration can be applied by Command Line Interface (CLI) options or as environment variable settings. When both environment variable and CLI argument are provided, the latest will overrule the env variable.
```

Comment:
"When both **the** environment variable and **the** CLI argument are provided, the latest..."

Reply:
fixed